src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +564,15 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        if (self::$constantSupport) {
+                            return @constant(substr($scalar, 11));
```

Comment:
It would be consistent with the current `XmlLoader` indeed. But `null` will be return anyway with an `E_WARNING`.